Stellar-overlay.x

```diff
@@ -94,7 +94,6 @@ enum MessageType
     AUTH = 2,
     DONT_HAVE = 3,
 
-    GET_PEERS = 4, // gets a list of peers this guy knows about
```

Comment:
Good idea! I left a comment that GET_PEERS is deprecated. I did not mention not re-using it, since at some point it would be safe to re-use it (once overlay drops support for GET_PEERS completely, for example)